### PR TITLE
Cubrid Plugin Extension

### PR DESCRIPTION
**Pull request by longhaseng52**
- Driver download from maven repository
- Connection properties
- Create folders to categorized by user
- Display list of user
- Display table based on user
- Display system table
- Display view based on user
- Display system view

### CODE REVIEW

**Comment by hwany7seo** [comment]
Hello @longhaseng52  
PR is really great. But I have a list that I need to check.
would like you to check list.

[check list]
1.  Exception on Navigator -> Create Table
image

2.  System Table&View should only be  visible in 'DBA' user

3. In Server 11.2 and later, column in tables appear duplicated.
ex) dba.test2
CREATE TABLE [dba].[test2] (
	[aaa] CHARACTER(1),
	[bbb] CHARACTER(1),
	[ggg] INTEGER
)
public.test2
CREATE TABLE [public].[test2] ([bbb] INTEGER)
image

4. If a procedure is not retrieved, it must be removed from the list

**Comment by rathana-pvs** [comment]
> 3. In Server 11.2 and later, column in tables appear duplicated.
>    ex) dba.test2
>    CREATE TABLE [dba].[test2] (
>    [aaa] CHARACTER(1),
>    [bbb] CHARACTER(1),
>    [ggg] INTEGER
>    )
>    public.test2
>    CREATE TABLE [public].[test2] ([bbb] INTEGER)
>    image

We have identified this issue. This issue is related to TableColumn. to fix this issue we need to modify some  of column. 
My opinion is: to avoid duplicate task, Can we fix this issue on third PR? 
because in the third PR we are going to focus column implementation.

**Comment by hwany7seo** [comment]
> We have identified this issue. This issue is related to TableColumn. to fix this issue we need to modify some of column. My opinion is: to avoid duplicate task, Can we fix this issue on third PR? because in the third PR we are going to focus column implementation.

I wish it was included now, but I'll accept your opinion if so.

**Comment by longhaseng52** [comment]
Dear @hwany7seo, we have fixed all the issues above including "column in tables appear duplicated". Please kindly check if there is still any issues. Thank you.

File: plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/edit/CubridTableManager.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import org.jkiss.dbeaver.model.struct.DBSObject;
+import org.jkiss.dbeaver.model.struct.cache.DBSObjectCache;
+import org.jkiss.utils.CommonUtils;
+
+public class CubridTableManager extends SQLTableManager<CubridTableBase, CubridObjectContainer> {
+
+	private static final String NEW_TABLE_NAME = "NEW_TABLE";
+	private static final String TABLE_TYPE_TABLE = "TABLE";
+	private static final Class<? extends DBSObject>[] CHILD_TYPES = CommonUtils.array(
+	        GenericTableColumn.class,
```
Comment (hwany7seo) [inline plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/edit/CubridTableManager.java]: Shouldn’t Generic... be CUBRID...?

**Review by hwany7seo — COMMENTED** [review]
Other issues have been resolved. But a added new issue.

[New Issue]

1.  Columns information can be viewed in 'Navigator', 'Properties' editor  And on Server 11.0
Then, press F5 to refresh and it will be displayed.

test) Click Columns or Double Click 'athlete' in Navigator List

2.  When the table name(DBA.test2, PUBLIC.test2) is the same, PK, FK, and INDEX information are shared. ON Server 11.2

ex) 
CREATE TABLE DBA.[test2] (
	[aaa] CHARACTER(1),
	[bbb] CHARACTER(1),
	[ggg] INTEGER NOT NULL,
	[custkey] INTEGER,
	CONSTRAINT [pk] PRIMARY KEY([ggg]),
	INDEX [i_test2_bbb] ([bbb]),
	CONSTRAINT [fk_custkey] FOREIGN KEY([custkey]) REFERENCES [dba.customer] ([c_custkey]) ON DELETE RESTRICT ON UPDATE RESTRICT
)

CREATE TABLE PUBLIC.[test2] 
(
	[bbb] INTEGER
)

**Review by hwany7seo — APPROVED** [review]
Thank you for hard works.

**Comment by kisoo-han** [comment]
@longhaseng52 ,
I'm also appreciate that. I checked both issues work fine.

**Pull request merged**